clients/tabby-agent/src/codeCompletion/index.ts

```diff
@@ -489,7 +514,7 @@ export class CompletionProvider implements Feature {
     if (solution) {
       this.completionStats.addProviderStatsEntry({ triggerMode: request.manually ? "manual" : "auto" });
       this.logger.info(`Completed processing completions, choices returned: ${solution.items.length}.`);
-      this.logger.trace("Completion solution:", { solution: solution.toInlineCompletionList() });
+      this.logger.trace("Completion solution:" + JSON.stringify({ solution: solution.toInlineCompletionList() }));
```

Comment:
This was my mistake; I changed TRACE to the explicit log level INFO because there was too much extra information for me at the TRACE level. When I changed it back, I only updated the method.